app/models/cd_api/defendant.rb

```diff
@@ -2,7 +2,7 @@
 
 module CdApi
   class Defendant < BaseModel
-    has_many :offence_summary, class_name: 'cd_api/offence_summary'
+    has_many :offence_summaries, class_name: 'cd_api/offence_summary'
```

Comment:
great spot, 
what about adding a small test in the  
`models/cd_api/defendant_spec.rb` ?